drivers/include/periph/hwrng.h

```diff
@@ -59,6 +62,13 @@ void hwrng_init(void);
  */
 void hwrng_read(void *buf, unsigned int num);
 
+#if defined(MODULE_VFS) || defined(DOXYGEN)
+/**
+ * @brief urandom driver for vfs
+ */
+extern const vfs_file_ops_t hwrng_vfs_ops;
```

Comment:
This should go in its own header. 
Otherwise I am afraid that we might be getting the code a bit too tangled up.